src/App.tsx

```diff
@@ -26,70 +29,132 @@ type ReorderOptions = {
   isReversed: boolean,
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  visibleGoods.sort(
+    (prevGood, nextGood) => {
+      switch (sortType) {
+        case SortType.LENGTH:
+          return prevGood.length - nextGood.length;
+        case SortType.ALPHABET:
+          return prevGood.localeCompare(nextGood);
+        default:
+          return 0;
+      }
+    },
+  );
 
-  return visibleGoods;
+  return isReversed
+    ? visibleGoods.reverse()
+    : visibleGoods;
```

Comment:
u you can reduce the complexity and make reverse inside sort function)

Reply:
`  return visibleGoods.sort(
    (prevGood, nextGood) => {
      switch (sortType) {
        case SortType.LENGTH:
          return isReversed
            ? nextGood.length - prevGood.length
            : prevGood.length - nextGood.length;
        case SortType.ALPHABET:
          return isReversed
            ? nextGood.localeCompare(prevGood)
            : prevGood.localeCompare(nextGood);
        case SortType.NONE:
          return isReversed ? -1 : 0;
        default:
          return 0;
      }
    },
  );`
  
  done
  effectiveness increased
  complexity reduced twice
  but code became a little less understandable
  
 but technical aspects are more important /// so agreed with your advice